mobile/library/common/jni/jni_interface.cc

```diff
@@ -20,12 +21,13 @@ using Envoy::Platform::EngineBuilder;
 
 JNIEXPORT jint JNICALL JNI_OnLoad(JavaVM* vm, void* reserved) {
   JNIEnv* env = nullptr;
-  if (vm->GetEnv(reinterpret_cast<void**>(&env), JNI_VERSION) != JNI_OK) {
+  if (vm->GetEnv(reinterpret_cast<void**>(&env), Envoy::JNI::JavaVirtualMachine::getJNIVersion()) !=
```

Comment:
optional nit but if this is the only use case it might be easier to make this isJniVersionOk just to keep the code simpler

Reply:
updated per your suggestion